app/json_df_options.txt
"{\"columns\":[\"HF_Name\",\"CUSIP\",\"Company\",\"Value\",\"Shares\",\"Option_Type\"],\"index\":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67],\"data\":[[\"ANSON FUNDS\",\"01609W102\",\"ALIBABA GROUP HLDG LTD\",3990,17600,\"Call\"],[\"ANSON FUNDS\",\"067901108\",\"BARRICK GOLD CORP\",6233,250000,\"Call\"],[\"ANSON FUNDS\",\"15135U109\",\"CENOVUS ENERGY INC\",1888,200000,\"Call\"],[\"ANSON FUNDS\",\"292104106\",\"EMPIRE ST RLTY TR INC\",1113,100000,\"Call\"],[\"ANSON FUNDS\",\"26884U109\",\"EPR PPTYS\",1188,25500,\"Call\"],[\"ANSON FUNDS\",\"26922A842\",\"ETF SER SOLUTIONS\",2691,100000,\"Put\"],[\"ANSON FUNDS\",\"464287432\",\"ISHARES TR\",13545,100000,\"Call\"],[\"ANSON FUNDS\",\"464287655\",\"ISHARES TR\",2298,10400,\"Put\"],[\"ANSON FUNDS\",\"464287739\",\"ISHARES TR\",2905,31600,\"Put\"],[\"ANSON FUNDS\",\"554382101\",\"MACERICH CO\",1170,100000,\"Call\"],[\"ANSON FUNDS\",\"78462F103\",\"SPDR S&P 500 ETF TR\",6341,16000,\"Put\"],[\"ANSON FUNDS\",\"878742204\",\"TECK RESOURCES LTD\",2407,100000,\"Call\"],[\"ANSON FUNDS\",\"88034P109\",\"TENCENT MUSIC ENTMT GROUP\",2561,125000,\"Call\"],[\"ANSON FUNDS\",\"917286205\",\"URSTADT BIDDLE PPTYS INC\",882,53000,\"Call\"],[\"ANSON FUNDS\",\"92556H206\",\"VIACOMCBS INC\",4798,106400,\"Call\"],[\"ANSON FUNDS\",\"984017103\",\"XENIA HOTELS & RESORTS INC\",624,32000,\"Call\"],[\"HOUND PARTNERS, LLC\",\"36467W109\",\"GAMESTOP CORP NEW\",14085,74200,\"Call\"],[\"HOUND PARTNERS, LLC\",\"36467W109\",\"GAMESTOP CORP NEW\",7878,41500,\"Put\"],[\"HOUND PARTNERS, LLC\",\"36467W109\",\"GAMESTOP CORP NEW\",37964,200000,\"Put\"],[\"HOUND PARTNERS, LLC\",\"86646P103\",\"SUMO LOGIC INC\",4244,225000,\"Call\"],[\"HOUND PARTNERS, LLC\",\"890516107\",\"TOOTSIE ROLL INDS INC\",9787,295400,\"Call\"],[\"HOUND PARTNERS, LLC\",\"890516107\",\"TOOTSIE ROLL INDS INC\",7623,230100,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"013872106\",\"ALCOA CORP\",32490,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"10922N103\",\"BRIGHTHOUSE FINL INC\",22125,500000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"172967424\",\"CITIGROUP INC\",72750,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"172967424\",\"CITIGROUP INC\",72750,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"172967424\",\"CITIGROUP INC\",72750,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"172967424\",\"CITIGROUP INC\",72750,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"185899101\",\"CLEVELAND-CLIFFS INC NEW\",20110,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"185899101\",\"CLEVELAND-CLIFFS INC NEW\",20110,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"247361702\",\"DELTA AIR LINES INC DEL\",16898,350000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"247361702\",\"DELTA AIR LINES INC DEL\",48280,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"D18190898\",\"DEUTSCHE BANK A G\",12000,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"D18190898\",\"DEUTSCHE BANK A G\",12000,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"D18190898\",\"DEUTSCHE BANK A G\",12000,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"D18190898\",\"DEUTSCHE BANK A G\",24000,2000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"31816Q101\",\"FIREEYE INC\",19570,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"35671D857\",\"FREEPORT-MCMORAN INC\",32930,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"369604103\",\"GENERAL ELECTRIC CO\",13130,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"37045V100\",\"GENERAL MTRS CO\",57460,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"37045V100\",\"GENERAL MTRS CO\",57460,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"375558103\",\"GILEAD SCIENCES INC\",64630,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"458140100\",\"INTEL CORP\",64000,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"61945C103\",\"MOSAIC CO NEW\",19756,625000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"61945C103\",\"MOSAIC CO NEW\",31610,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"62886E108\",\"NCR CORP NEW\",37950,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"62955J103\",\"NOV INC\",41160,3000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"670346105\",\"NUCOR CORP\",24081,300000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"68389X105\",\"ORACLE CORP\",28068,400000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"724479100\",\"PITNEY BOWES INC\",7007,850400,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"767204100\",\"RIO TINTO PLC\",77650,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"806857108\",\"SCHLUMBERGER LTD\",27190,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"806857108\",\"SCHLUMBERGER LTD\",27190,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"78462F103\",\"SPDR S&P 500 ETF TR\",396330,1000000,\"Put\"],[\"MASTERS CAPITAL MANAGEMENT\",\"78462F103\",\"SPDR S&P 500 ETF TR\",396330,1000000,\"Put\"],[\"MASTERS CAPITAL MANAGEMENT\",\"878742204\",\"TECK RESOURCES LTD\",9590,500000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"H8817H100\",\"TRANSOCEAN LTD\",3550,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"910047109\",\"UNITED AIRLS HLDGS INC\",40278,700000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"910047109\",\"UNITED AIRLS HLDGS INC\",57540,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"911312106\",\"UNITED PARCEL SERVICE INC\",169990,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"912909108\",\"UNITED STATES STL CORP NEW\",26170,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"92556V106\",\"VIATRIS INC\",13970,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"949746101\",\"WELLS FARGO CO NEW\",39070,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"949746101\",\"WELLS FARGO CO NEW\",78140,2000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"949746101\",\"WELLS FARGO CO NEW\",39070,1000000,\"Call\"],[\"MASTERS CAPITAL MANAGEMENT\",\"949746101\",\"WELLS FARGO CO NEW\",39070,1000000,\"Call\"],[\"SYMPHONY ASSET MANAGEMENT LLC\",\"464287655\",\"ISHARES RUSSELL 2000 ETF\",4295,30000,\"Put\"],[\"SYMPHONY ASSET MANAGEMENT LLC\",\"78462F103\",\"SPDR S&P 500 ETF TRUST\",10793,35000,\"Put\"]]}"

app/config.py
import os

blog_1_url = "http://www.stockfries.com/blog/1_gambling"
blog_2_url = "http://www.stockfries.com/blog/2_basics"


change_threshold = 50000

ownership_colors = [
    'rgba(0,0,255,'+str(i/100)+')' for i in range(2, 42, 2)[::-1]]
positives_colors = [
    'rgba(0,255,0,'+str(i/100)+')' for i in range(2, 42, 2)[::-1]]
negatives_colors = [
    'rgba(255,0,0,'+str(i/100)+')' for i in range(2, 42, 2)[::-1]]

figi_api_key = '34dada0c-5a22-4f5a-a095-739afe8fe591' 

scrapped_json_fn = 'json_df.txt'
scrapped_json_fn_no_ticker = 'json_df_no_ticker.txt'
scrapped_json_fn_hedgefund = 'json_df_dataByHedgeFund.txt'
scrapped_json_fn_options = 'json_df_options.txt'

fund_dict = {'ABRAMS CAPITAL MANAGEMENT, L.P.': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1358706&owner=exclude&count=40&hidefilings=0',
             'AKRE CAPITAL MANAGEMENT': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1112520&owner=exclude&count=40&hidefilings=0',
             'ANSON FUNDS': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1491072&owner=exclude&count=40&hidefilings=0',
             'APPALOOSA LP': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1656456&owner=exclude&count=40&hidefilings=0',
             'ARLINGTON VALUE CAPITAL, LLC': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1568820&owner=exclude&count=40&hidefilings=0',
             'ATLANTIC INVESTMENT MANAGEMENT, INC.': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1063296&owner=exclude&count=40&hidefilings=0',
             'BAKER BROS. ADVISORS LP': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1263508&owner=exclude&count=40&hidefilings=0',
             'BOGLE INVESTMENT MANAGEMENT L P':'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=0001131667&owner=include&count=40&hidefilings=0',
             'BLOOMBERGSEN U.S. FUND LP': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1641744&owner=exclude&count=40&hidefilings=0',
             'CAPITAL FUND MANAGEMENT': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1323645&owner=exclude&count=40&hidefilings=0',
             'CANYON CAPITAL ADVISORS LLC':'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=0001074034&owner=include&count=40&hidefilings=0',
             'COATUE MANAGEMENT LLC': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1135730&owner=exclude&count=40&hidefilings=0',
             'CONTRARIAN CAPITAL MANAGEMENT': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1050417&owner=exclude&count=40&hidefilings=0',
             'CORVEX MANAGEMENT LP': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1535472&owner=exclude&count=40&hidefilings=0',
             'CRABEL CAPITAL MANAGEMENT, LLC':'https://www.sec.gov/cgi-bin/browse-edgar?company=Crabel&match=&filenum=&State=&Country=&SIC=&myowner=exclude&action=getcompany',
             'DALAL STREET, LLC': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1549575&owner=exclude&count=40&hidefilings=0',
             'DORSAL CAPITAL MANAGEMENT': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1547007&owner=exclude&count=40&hidefilings=0',
             'ELLIOTT MANAGEMENT CORP': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1048445&owner=exclude&count=40&hidefilings=0',
             'EMINENCE CAPITAL, LP': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1107310&owner=exclude&count=40&hidefilings=0',
             'GATES CAPITAL MANAGEMENT, INC': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1312908&owner=exclude&count=40&hidefilings=0',
             'GLENVIEW CAPITAL MANAGEMENT, LLC': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1138995&owner=exclude&count=40&hidefilings=0',
             'HAWK RIDGE CAPITAL MANAGEMENT': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1609074&owner=exclude&count=40&hidefilings=0',
             'HARVEST CAPITAL STRATEGIES': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1226355&owner=exclude&count=40&hidefilings=0',
             'HIGHBRIDGE CAPITAL MANAGEMENT LLC':'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=0000919185&owner=include&count=40&hidefilings=0',
             'HOUND PARTNERS, LLC': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1353316&owner=exclude&count=40&hidefilings=0',
             'ICAHN CARL C': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=921669&owner=exclude&count=40&hidefilings=0',
             'III Capital Management ':'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=0001783139&owner=include&count=40&hidefilings=0',
             'JANA PARTNERS LLC': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1159159&owner=exclude&count=40&hidefilings=0',
             'JOHO CAPITAL LLC': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1106500&owner=exclude&count=40&hidefilings=0',
             'LONE PINE CAPITAL LLC': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1061165&owner=exclude&count=40&hidefilings=0',
             'LONG POND CAPITAL, LP': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1499066&owner=exclude&count=40&hidefilings=0',
             'MASTERS CAPITAL MANAGEMENT': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1104186&owner=exclude&count=40&hidefilings=0',
             'MANGROVE PARTNERS': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1535392&owner=exclude&count=40&hidefilings=0',
             'MARCATO CAPITAL MANAGEMENT LP': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1541996&owner=exclude&count=40&hidefilings=0',
             'Melvin Capital Management LP': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=0001628110&owner=include&count=40&hidefilings=0',
             'MKP Capital Management, L.L.C.':'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=0001534949&owner=include&count=40&hidefilings=0',
             'MIG CAPITAL': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1425649&owner=exclude&count=40&hidefilings=0',
             'OMEGA ADVISORS INC.': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=898202&owner=exclude&count=40&hidefilings=0',
             'PARAMETRICA ASSET MANAGEMENT LTD': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1535090&owner=exclude&count=40&hidefilings=0',
             'PENNANT CAPITAL MANAGEMENT, LLC': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1168664&owner=exclude&count=40&hidefilings=0',
             'PERRY CAPITAL': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=919085&owner=exclude&count=40&hidefilings=0',
             'PERSHING SQUARE CAPITAL MANAGEMENT, L.P.': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1336528&owner=exclude&count=40&hidefilings=0',
             'POLAR ASSET MANAGEMENT PARTNERS': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1326389&owner=exclude&count=40&hidefilings=0',
             'RA CAPITAL MANAGEMENT, LLC': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1346824&owner=exclude&count=40&hidefilings=0',
             'RUANE, CUNNIFF & GOLDFARB INC': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=85624&owner=exclude&count=40&hidefilings=0',
             'SEGANTII': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1544676&owner=exclude&count=40&hidefilings=0',
             'SOROS FUND MANAGEMENT LLC': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1029160&owner=exclude&count=40&hidefilings=0',
             'SOUTHEASTERN ASSET MANAGEMENT INC': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=807985&owner=exclude&count=40&hidefilings=0',
             'SPO ADVISORY CORP': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=919468&owner=exclude&count=40&hidefilings=0',
             'STARBOARD VALUE LP': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1517137&owner=exclude&count=40&hidefilings=0',
             'SYMPHONY ASSET MANAGEMENT LLC': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=0001030815&owner=include&count=40&hidefilings=0',
             'TCI FUND MANAGEMENT LTD': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1647251&owner=exclude&count=40&hidefilings=0',
             'TIDE POINT': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1590569&owner=exclude&count=40&hidefilings=0',
             'TIGER GLOBAL MANAGEMENT LLC': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1167483&owner=exclude&count=40&hidefilings=0',
             'TRIAN FUND MANAGEMENT, L.P.': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1345471&owner=exclude&count=40&hidefilings=0',
             'VIKING GLOBAL INVESTORS LP': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1103804&owner=exclude&count=40&hidefilings=0',
             'VR ADVISORY SERVICES': 'https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=1348145&owner=exclude&count=40&hidefilings=0'}
